### PR TITLE
Implement ‘Cloud tag’, ‘Navigation Bar’

### PR DESCRIPTION
**Pull request by MechanicKim**
I think that these seem to require more supplementation.
I will cover them next PR.

### CODE REVIEW

File: src/components/userProfile/CloudTag.js [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+  orderData() {
+    let result = [];
+    let tagList = this.props.tagList;
+    tagList.sort((a, b) => {
+      if (a.point === b.point) return 0;
+      else return a.point > b.point ? -1 : 1;
+    });
+
+    const maxPoint = tagList[0].point;
+    let flag = true;
```
Comment (ifndefdeadmau5) [inline src/components/userProfile/CloudTag.js]: If you leave comment about this 'flag' or rename to well explained name, will be better

File: src/components/userProfile/UserOverview.js [hunk 2]
```diff
@@ -150,38 +151,15 @@ class UserOverview extends Component {
...
-              (
-                <Text
-                  key={index}
-                  style={[fontStyle[scoreArray[index]], styles.personality]}>
-                  {value}
-                </Text>
-              ))
-          }
-        </View>
+        <CloudTag tagList={tagList} width={250} />
```
Comment (ifndefdeadmau5) [inline src/components/userProfile/UserOverview.js:162]: Is width always fixed like 250?

File: src/components/userProfile/UserProfile.js [hunk 3]
```diff
@@ -43,91 +47,24 @@ class UserProfile extends Component {
...
-        resolve(result.bookmarked);
-      }
-    });
-  }
 
   getProfileImage(status) {
-    let image;
     if (status.profile_picture) {
-      image = {
+      const Image = {
```
Comment (ifndefdeadmau5) [inline src/components/userProfile/UserProfile.js:61]: Why first letter has changed to upper?

**Review by ifndefdeadmau5 — COMMENTED** [review]
This PR is awesome 👍  Please check few comment

File: src/components/userProfile/CloudTag.js (hunk 1 above)
Comment (MechanicKim) [reply src/components/userProfile/CloudTag.js]: Okay! I'll rename it. Thanks!

File: src/components/userProfile/UserOverview.js (hunk 2 above)
Comment (MechanicKim) [reply src/components/userProfile/UserOverview.js:162]: Yes! Total width is fixed.
If this size is not reasonable, it is available to change this size.

File: src/components/userProfile/UserProfile.js (hunk 3 above)
Comment (MechanicKim) [reply src/components/userProfile/UserProfile.js:61]: Image can be imported. I already used this expression. For example,
`import PatternBackground from '../../resources/pattern.png';`
I thought that image is like a component. So I used uppercase.

Comment (ifndefdeadmau5) [reply src/components/userProfile/UserProfile.js:61]: Okay. got it!

**Comment by ifndefdeadmau5** [comment]
LGTM

[Approved with PullApprove](https://pullapprove.com/sv-bootcamp/bridgeme-client/pull-request/171/?utm_source=github-pr&utm_medium=comment-badge&utm_campaign=sv-bootcamp/bridgeme-client)

**Pull request merged**